src/memory-model.md

```diff
@@ -1,5 +1,45 @@
 # Memory model
 
-Rust does not yet have a defined memory model. Various academics and industry professionals
-are working on various proposals, but for now, this is an under-defined place
-in the language.
+r[memory]
+
+The Memory Model of Rust is incomplete and not fully decided. The following is some of the detail worked out so far.
+
+## Bytes
+
+r[memory.byte]
+
+r[memory.byte.intro]
+The most basic unit of memory in Rust is a byte. All values in Rust are computed from 0 or more bytes read from an allocation.
```

Comment:
Is "value" defined anywhere?

Reply:
Values are defined constructively, with each "class" of types. As mentioned in another comment, these are present in different chapters at the request of T-lang-doc and T-spec.